README.md

```diff
@@ -12,6 +12,45 @@
 
 ## Dataset
  * [Credit card fraud](https://www.kaggle.com/dalpozz/creditcardfraud) 
+ * Add three fake columns to the original dataset: unique user ID, user type (international or domestic), unique account number
+ * Columns appended will be used in aggregation in application.
+ * Split the new dataset "creditcardfake" into training and test sets.
+ * [Download datasets](https://drive.google.com/open?id=1QIeiHcDd0JGeK8jWN-GQskRY-3lvG9R8)
+ 
+## Model
+ * Fraud detection model using logistics regression
+ 
+## Kafka Stories and Introduction
+Apache Kafka is a high-throughput distributed streaming platform and messaging system developed by the Apache Software Foundation written in Scala and Java. Kafka is implemented as a distributed commit log.
```

Comment:
do we need a description of what a distributed commit log is ?

Reply:
Added a paragraph to explain.